crates/volta-core/src/tool/node.rs

```diff
@@ -13,11 +15,21 @@ where
     session.add_event_start(ActivityKind::Node);
 
     match session.current_platform()? {
-        Some(ref platform) => {
+        Some(platform) => {
+            match platform.source() {
+                Source::Project => {
+                    debug!("Using node@{} from project configuration", platform.node())
```

Comment:
Nit: it'd be nice to use the `style::tool_version` formatter here:

```rust
debug!("Using {} from project configuration", tool_version("node", platform.node()));
```

As an aside, I'd be inclined to do this to avoid having to tweak the messages separately if we reword in the future:

```rust
let source = match platform.source() {
    Source::Project => "project",
    Source::User => "default",
};
let version = tool_version("node", platform.node());
debug!("Using {} from {} configuration", version, source);
```

Reply:
Good call in both cases! Updated!